addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1771,7 +1771,7 @@ const VisibilityPageOptionUpdate = options.Class.extend({
         // toggled (otherwise it would be about editing an element which
         // is actually never displayed on the page).
         const widget = this._requestUserValueWidgets(this.showOptionWidgetName)[0];
-        widget.$el.click();
+        widget.notifyValueChange(false);
```

Comment:
Mmh, this actually works by chance. `notifyValueChange` is only supposed to notify that the value has changed as its name indicates... but here it actually simulate the change too. This is because we still have some code which forces a preview when simulating a non preview like you do here...

I prefer to keep a hack we know we have to improve rather than relying on luck: use `widget.$('we-checkbox').click()` like you suggested in the first place, as a 14.0 fix :+1: